packages/helix-shared-config/src/ConfigValidator.js

```diff
@@ -31,40 +32,24 @@ const schemas = [
   /* eslint-enable global-require */
 ];
 
-class ValidationError extends Error {
+class HelixConfigValidationError extends ValidationError {
   constructor(msg, errors = []) {
-    function prettyname(path, schema) {
-      if (path.startsWith('.strains')) {
-        return `${schema.title || 'Invalid Strain'} ${path.replace(/\.strains(\.|\[')(.*)/, '$2').replace(/'.*/, '')}`;
-      }
-      return `${schema.title || schema.$id} ${path}`;
-    }
-
-    const detail = errors.map(({
-      keyword, dataPath, message, data, params, parentSchema,
-    }) => {
-      if (keyword === 'additionalProperties') {
-        return `${prettyname(dataPath, parentSchema)} has unknown property '${params.additionalProperty}'`;
-      }
-      if (keyword === 'required' && dataPath === '') {
-        return 'A set of strains and a default strain are missing.';
-      }
-      if (keyword === 'required' && dataPath === '.strains') {
-        return 'A default strain is missing.';
-      }
-      if (keyword === 'required') {
-        return `${prettyname(dataPath, parentSchema)} ${message}`;
-      }
-      if (keyword === 'oneOf' && dataPath.startsWith('.strains')) {
-        return `${prettyname(dataPath, parentSchema)} must be either a Runtime Strain or a Proxy Strain`;
-      }
-      return `${prettyname(dataPath, parentSchema)} ${message}: ${keyword}(${JSON.stringify(data)}, ${JSON.stringify(params)})`;
-    }).join('\n');
-    super(`Invalid configuration:
-${detail}
+    super(msg, errors, HelixConfigValidationError.mapError);
+  }
 
-${msg}`);
-    this._errors = errors;
+  static mapError({
+    keyword, dataPath, message, data, params, parentSchema,
+  }) {
+    if (keyword === 'required' && dataPath === '') {
+      return 'A set of strains and a default strain are missing.';
+    }
+    if (keyword === 'required' && dataPath === '.strains') {
+      return 'A default strain is missing.';
+    }
+    if (keyword === 'oneOf' && dataPath.startsWith('.strains')) {
+      return `${ValidationError.prettyname(dataPath, parentSchema)} must be either a Runtime Strain or a Proxy Strain`;
+    }
+    return ValidationError.mapError(keyword, dataPath, message, data, params, parentSchema);
```

Comment:
call `super.mapError` although it is not possible to call it super, as `mapError` is called in the constructor (see comment in `ValidationError` constructor)